tests/conftest.py

```diff
@@ -45,6 +45,6 @@ def _cleanup_temporary_dirs(request):
         cleanup_temp_dirs()
     else:
         print(
-            f"NOT cleaning up {len(active_temp_dirs())} temp dirs to ease troubleshooting"
```

Comment:
by the way, I think the `_cleanup_temporary_dirs` fixture should have a `scope="session"`, not "function" so they'll be torn down at the end of the session, not at the end of each test function, no?
https://docs.pytest.org/en/6.2.x/fixture.html#fixture-scopes

Reply:
Maybe I'm missing something obvious but ... why do I want them to last to the end of the session as opposed to being destroyed at the end of the test? - they shouldn't ever share across tests so we're done with them at the end of the test.

Reply:
Resolving for now, plz reopen if I need to change it